lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.h

```diff
@@ -384,6 +392,9 @@ class ObjectFileELF : public lldb_private::ObjectFile {
   /// ELF dependent module dump routine.
   void DumpDependentModules(lldb_private::Stream *s);
 
+  /// ELF dump the .dynamic section
+  void DumpELFDynamic(lldb_private::Stream *s);
```

Comment:
Can this take a `Stream &` instead of a `Stream *`? The implementation assumes that the pointer is valid, a reference would give us a better guarantee at compile time.

Reply:
All other dump calls use a "Stream *" so I mirrored those. I can make a separate patch to switch these over to "Stream &". Would that be ok? I didn't want to make the patch bigger

Reply:
I wouldn't say its required, but I'd like changing these to a reference (in another patch). If the patch is trivial/obvious, i don't think it requires a review.